sirepo/raydata/scan_monitor.py

```diff
@@ -361,11 +364,21 @@ async def _process_analysis_queue():
                 ) as l:
                     try:
                         for n in d.get_notebooks():
+                            pkjinja.render_resource(
+                                _PAPERMILL_SCRIPT,
+                                PKDict(
+                                    input_f=n.input_f,
+                                    output_f=n.output_f,
+                                    papermill_args=" ".join(d.get_papermill_args()),
+                                    conda_prefix=cfg.conda_prefix,
+                                    chx_conda_env=cfg.chx_conda_env,
```

Comment:
Rather than having variables specific to a beamline in this generic code we should move the construction of the script into the analysis_driver for the beamline. That was csx's script won't have any mention of chx and we won't have mention of chx in scan_monitory.py

Reply:
> csx's script won't have any mention of chx

Are you thinking that each analysis driver should have its own jinja template? It seems like there will be a lot of repeated code if we start using a conda env for csx as well.

Reply:
I am. I agree there will be repeated code. I think there are 2 options. Choose whichever you like more:
1. Separate out for now and we can create an abstraction when the time comes.
2. Create the abstraction now. There shouldn't be mention of words like "chx" in any "general" files (ex scan_monitor.py). Anything specific to a beamline should live in the analysis driver for the beamline (ex the name of the conda env for chx).

Reply:
cd9819208e9555b3aef1a844e7a669af3047a86e